core/ir/x86/decode_table.c

```diff
@@ -7075,8 +7079,8 @@ const instr_info_t rm_extensions[][8] = {
     {OP_enclu,  0xd70f0172, catUncategorized, "enclu", eax, ebx, eax, ebx, ecx, mrm|xop, x, exop[0xff]},
   },
   { /* rm extension 5 */
-    {INVALID,   0x0f0131, catUncategorized, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
-    {INVALID,   0x0f0131, catUncategorized, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {OP_xsusldtrk, 0xf201e808, catUncategorized, "xsusldtrk", xx, xx, xx, xx, xx, reqp, x, END_LIST},
```

Comment:
Let's put in a category for any new instruction.  I guess these would be catOther but that seems better than uncategorized.

Reply:
Done.